src/main/java/com/es/phoneshop/model/product/service/CartServiceImpl.java

```diff
@@ -0,0 +1,66 @@
+package com.es.phoneshop.model.product.service;
+
+import com.es.phoneshop.model.product.bean.Cart;
+import com.es.phoneshop.model.product.bean.CartItem;
+import com.es.phoneshop.model.product.bean.Product;
+import com.es.phoneshop.model.product.dao.ArrayListProductDao;
+
+import javax.servlet.http.HttpSession;
+import java.text.NumberFormat;
+import java.text.ParseException;
+import java.util.List;
+
+public class CartServiceImpl implements CartService {
+    private static volatile CartService instance;
+    private final static String NOT_NUMBER = "Not a number";
```

Comment:
Constants generally declared at the top of the class state, check and fix for the entire project